Library/Formula/pam_u2f.rb

```diff
@@ -0,0 +1,38 @@
+class PamU2f < Formula
+  desc "Provides an easy way to use U2F-compliant authenticators with PAM."
+  homepage "https://developers.yubico.com/pam-u2f/"
```

Comment:
Only reason I did it this way was to model after pam_yubico. I can change though...what do you think?